.github/CODEOWNERS

```diff
@@ -5,3 +5,5 @@ internal/codespaces/ @cli/codespaces
 
 # Limit Package Security team ownership to the attestation command package
 pkg/cmd/attestation/ @cli/package-security
+
+pkg/cmd/attestation/verification/embed/tuf-repo.github.com/ @trevrosen @kommendorkapten
```

Comment:
Do you think maybe this should be managed by a team in entitlements?

Reply:
> Do you think maybe this should be managed by a team in entitlements?

Good idea, let me look into this.

Reply:
I've updated the PR so the new [tuf-root-reviewers team](https://github.com/orgs/cli/teams/tuf-root-reviewers) owns this directory.